### DISCUSSION TITLE
Solid.js inside react.js app

### DISCUSSION THREAD

**Discussion by potikhanovsergey** [vikejs/vike#591] [Discussion]
Can vite-ssr-plugin be configured to run solid.js inside react.js? I can run solid inside react but as soon as vite-ssr-plugin is added everything breaks

**Comment by samuelstroschein** [comment]
You need to elaborate on your question. Nobody will be able to help you if you don't state what errors you are getting.

**Comment by potikhanovsergey** [comment]
Actually solved this, nvm
